ramls/circulation.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Circulation
-version: v8.2
+version: v8.3
```

Comment:
Why has this version been changed when the circulation API hasn't changed?

Reply:
Mistype. Returned